src/client/lazy-app/Compress/Output/custom-els/TwoUp/index.ts

```diff
@@ -98,19 +98,24 @@ export default class TwoUp extends HTMLElement {
 
   // KeyDown event handler
   private _onKeyDown(event: KeyboardEvent) {
+    const target = event.target;
+    if (target instanceof HTMLElement && target.closest('input')) return;
```

Comment:
I'd probably just check target's `tagName`, it's a bit simpler and `input` doesn't have children anyway.